### DISCUSSION TITLE
xbel import on Legacy browsers

### DISCUSSION THREAD

**Discussion by Virgus** [floccusaddon/floccus#1207] [General]
For those as me who still use XP browsers (mainly MozillaLegacy/Palemoon forks as Serpent and NewMoon) there's no browser extension option and probably there won't be in the future. I've been searching for an export/import tool to translate the xbel file I got locally thanks to LoFloccus. It would be an easy way for manually synchronizing the newest browsers' bookmarks with the XP instances I use now and then.

I've spent almost two days searching the web and trying old and outdated tools. I managed to download and test all versions of the old xbelfox.xpi that I could find but none worked and I'm afraid I'm not skilled enough to fix that plugin.

Has anybody succeded in doing something similar ? Any xbel2html or xbel2json tool/script would work. Or maybe an xbel to places.sqlite converter if any exists somewhere...

Thanks,
V.

**Comment by marcelklehr** [comment]
Hey @Virgus 

there have been a few people asking similar questions, which lead me to work on #1180 which allows syncing via an HTML file, that you can easily import into any browser.

**Reply by Virgus** [reply]
Thanks, great news Marcel ! I'll try it asap.

**Reply by Virgus** [reply]
Hello Marcel, I had the time to test the html file feature finally and I both tried to set a storage file "bookmarks.html" and "bookmarks.htm" but both were written with XBEL format. I tried to read the code of #1180 to catch what kind of trigger activates the html format (I thought it was the file name extension...). What should I do to activate this feature exactly ? Am I missing something ? Thanks and have a nice day. V.

**Reply by marcelklehr** [reply]
Ah, there's no UI for this, yet. I've just been testing the backend side of this to make sure syncing works as expected.

**Reply by Virgus** [reply]
Thanks for this clarification. Looking forward to test next versions that might activate this useful feature 🙏
